index.html

```diff
@@ -314,6 +347,106 @@ <h3>Outline Button</h3>
         <section id="card-section">
           <div class="component-description">
             <h2>Card</h2>
+            <p>
+              cards are used to display content and action on a particular
+              topic.
+            </p>
+          </div>
+          <div class="component-example">
+            <h3>Vertical cards</h3>
+            <p>
+              It has 3 variants normal card, card with cancel button & card with
+              badge
+            </p>
+            <div class="component-example__container">
+              <div class="card-vertical">
+                <img src="./assets/images/card-img.png" alt="card" />
+                <div class="card-body">
+                  <h4 class="card-header">
+                    ASUS ROG Strix G15 Ryzen 9 Octa Core 5900HX - (16 GB/512 GB
+                    SSD/Windows 10 Home/4 GB Graphic...
+                  </h4>
+                  <p class="card-price">
+                    <span class="price">₹96,990 </span>
+                    <strike>₹1,36,990</strike>
+                  </p>
+                </div>
+                <div class="card-footer">
+                  <button class="btn btn-primary">Add to Cart</button>
+                  <i class="far fa-heart"></i>
+                  <i class="fas fa-share-alt"></i>
```

Comment:
Just a small suggestion, enclose icons inside the button or anchor tag so that you can add functionality to it when you are implementing javascript in the future.

Reply:
Oh yes, I'll do that! Thanks for the suggestion.